pom.xml

```
@@ -62,13 +62,12 @@
         <!-- The classPath setting is a workaround for https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=911925 -->
         <argLine>-Xmx1024m -XX:MaxPermSize=256m -Djdk.net.URLClassPath.disableClassPathURLCheck=true</argLine>
 
-        <beam.version>2.28.0</beam.version>
+        <beam.version>2.29.0</beam.version>
```

```suggestion
        <beam.version>2.31.0</beam.version>
```

Once 2.31 is released.